### ISSUE TITLE
github3.exceptions.AuthenticationFailed: 401 Requires authentication

### ISSUE THREAD

**Issue by MartinThoma** [jwass/geojsonio.py#14]
When I try to visualize a bigger geojson, I get:

```
Traceback (most recent call last):
  File "/home/math/.local/bin/geojsonio", line 11, in <module>
    sys.exit(main())
  File "/home/math/.local/lib/python3.5/site-packages/geojsonio/geojsonio.py", line 214, in main
    url = make_url(contents, args.domain)
  File "/home/math/.local/lib/python3.5/site-packages/geojsonio/geojsonio.py", line 93, in make_url
    gist = _make_gist(contents)
  File "/home/math/.local/lib/python3.5/site-packages/geojsonio/geojsonio.py", line 172, in _make_gist
    gist = ghapi.create_gist(description, files)
  File "/home/math/.local/lib/python3.5/site-packages/github3/decorators.py", line 37, in auth_wrapper
    raise error_for(r)
github3.exceptions.AuthenticationFailed: 401 Requires authentication
```

How can I authenticate?

**Comment by medeirosjoaquim** [comment]
It's the same problem described in issue #11 and fixed in #12.

**Comment by MartinThoma** [comment]
@medeirosjoaquim Thank you. Then I close it as it is a duplicate.
